README.md

```diff
@@ -1,2 +1,30 @@
 # liveDVR
 DVR implementation built on top of live streams
+
+# Installation #
+
+## Pre-requisites ##
+
+* Make sure node is installed
+* Install npm
+* Install nvm (curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.2/install.sh | bash)
```

Comment:
in production, QA, it's easier to deploy this way, so we don' need to upgrade node.js anymore, it's automatic using the .nvmrc . file